### ISSUE TITLE
Unable to add to/edit Neo field after updating craft version

### ISSUE THREAD

**Issue by TonyFwin** [spicywebau/craft-neo#310]
#### Description
Unable to add to/edit Neo field after updating craft version 3.4.10

#### Steps to reproduce
1. Update to craft 3.4.10
2. Create, edit, add fields to neo field and save
3. No changes are saved

#### Other information
- Neo version: 2.6.4
- Craft version: Craft CMS 3.4.10
- Is eager-loading used? No

**Comment by pvldigital** [comment]
Hi @TonyFwin,

I have replicated the issue and have started working on a fix.

**Comment by pvldigital** [comment]
@TonyFwin fix has been released in v2.6.5.

**Comment by jaydensmith** [comment]
@pvldigital I'm getting a new error with this release:
```
yii\base\ErrorException: Undefined index: wasModified in /Users/jaydensmith/xxx/vendor/spicyweb/craft-neo/src/Field.php:614
Stack trace:
#0 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/web/ErrorHandler.php(74): craft\web\ErrorHandler->handleError()
#1 /Users/jaydensmith/xxx/vendor/spicyweb/craft-neo/src/Field.php(614): craft\web\ErrorHandler->handleError()
#2 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/services/Fields.php(756): benf\neo\Field->beforeSave()
#3 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/controllers/FieldsController.php(272): craft\services\Fields->saveField()
#4 /Users/jaydensmith/xxx/vendor/yiisoft/yii2/base/InlineAction.php(57): craft\controllers\FieldsController->actionSaveField()
#5 /Users/jaydensmith/xxx/vendor/yiisoft/yii2/base/InlineAction.php(57): ::call_user_func_array:{/Users/jaydensmith/xxx/vendor/yiisoft/yii2/base/InlineAction.php:57}()
#6 /Users/jaydensmith/xxx/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams()
#7 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/web/Controller.php(178): craft\controllers\FieldsController->runAction()
#8 /Users/jaydensmith/xxx/vendor/yiisoft/yii2/base/Module.php(528): craft\controllers\FieldsController->runAction()
#9 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/web/Application.php(291): craft\web\Application->runAction()
#10 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/web/Application.php(559): craft\web\Application->runAction()
#11 /Users/jaydensmith/xxx/vendor/craftcms/cms/src/web/Application.php(270): craft\web\Application->_processActionRequest()
#12 /Users/jaydensmith/xxx/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest()
#13 /Users/jaydensmith/xxx/public_html/index.php(29): craft\web\Application->run()
#14 /Users/jaydensmith/.composer/vendor/weprovide/valet-plus/server.php(131): ::unknown()
#15 {main}
```

**Comment by pvldigital** [comment]
@jaydensmith oh i needed to check if that value is set. will release a hotfix.

**Comment by pvldigital** [comment]
@jaydensmith released a new version, 2.6.5.1. should fix that issue

**Comment by pvldigital** [comment]
@TonyFwin @jaydensmith It seems this issue was related to https://github.com/craftcms/cms/issues/5803. Since a fix for it has been made i'll be remove the temp fix that was implemented.
